README.rst

```diff
@@ -101,7 +101,8 @@ Snap (Snapcraft/Linux)
    snap install dvc --classic
 
 This corresponds to the latest tagged release.
-Add ``--edge`` for the latest ``master`` version.
+Add ``--beta`` for the latest tagged release candidate,
```

Comment:
Hmm I see double back quotes all over this doc, are those needed? Would be best to have consistency (all other docs I think use single ones)

Reply:
rst unfortunately isn't md. Python doc rendering ecosystems tend to work better with rst so I wouldn't recommend converting the readme to md (even though pypi has partial support for it now).

Also requested your review just in case you needed to update something in dvc.org

Reply:
Oh I didn't realize this is a rst file. OK

> Also requested your review just in case you needed to update something in dvc.org

- [ ] For this please open an issue or PR in the dvc.org repo if changes there are needed instead. You can probably check the install section to determine this.

Reply:
ok done https://github.com/iterative/dvc/pull/3865/ - good to merge now.

Reply:
Much appreciated! I'll take that over 🙂